plugins/contactpressuresensorsarray/include/yarp/dev/ContactPressureSensorsArrayDriver.h

```diff
@@ -0,0 +1,216 @@
+ * with respect to the link origin. Finally, the transformed equivalent wrench is mapped to an arrary of contact normal forces acting
+ * at the load cell locations. 
+ * 
+ * This mapping problem is an infinite solution problem and is solved by obtaining a pseudo inverse, 
+ * which gives the results with a minimized norm.
+ * 
+ * The configuation file must be specified in the sdf within the plugin element within the yarpConfigurationFile tags
+ * The configuration file must contain two groups [WRAPPER] and [DRIVER]
+ * Parameters accepted in the config argument of the open method for [WRAPPER]:
+ * | Parameter name| Type |Units|Default Value|Required |                 Description                |              Notes             |
+ * |:-------------:|:----:|:---:|:-----------:|:-------:|:------------------------------------------:|:------------------------------:|
+ * | name          |string|  -  |    -        |   Yes   | full name of the port opened by the device | MUST start with a '/' character|
+ * | period        |double|  ms |    20       |   No    | refresh period of broadcast value in ms    |    optional, default 20ms      |   
+ * | device        |string|  -  |    -        |   Yes   | name of the network wrapper to connect to  | MUST be set to analogServer    |    
+ *
+ * Parameters accepted in the config argument of the open method for [DRIVER]:
+ * | Parameter name|      Type     |Units|Default Value|Required |                       Description                      |                        Notes                     |
+ * |:-------------:|:-------------:|:---:|:-----------:|:-------:|:------------------------------------------------------:|:------------------------------------------------:|
+ * | device        |    string     |  -  |    -        |   Yes   | name of the device driver to instantiate               | MUST be set to gazebo_contactpressuresensorsarray|
+ * | loadCellNames |list of strings|  -  |    -        |   Yes   | list of load cell names, eg. (lr rr lf rf)             | locations MUST be obtained manually from urdf/sdf|
+ * | loadCellX     |list of doubles|  m  |    -        |   Yes   | list of load cell location X coordinate wrt link origin| MUST be the same size and order as loadCellName  |
+ * | loadCellY     |list of doubles|  m  |    -        |   Yes   | list of load cell location Y coordinate wrt link origin| MUST be the same size and order as loadCellName  |
+ * | loadCellZ     |list of doubles|  m  |    -        |   Yes   | list of load cell location Z coordinate wrt link origin| MUST be the same size and order as loadCellName  |
+ * | linkName      |     string    |  -  |    -        |   Yes   | name of the link associated to the contact sensor      |                            -                     |
+ *
+ * Implemented as a model plugin instead of a sensor plugin. This design choice was due to the fact that transformations related
```

Comment:
The phrase  "Implemented as a model plugin instead of a sensor plugin. " is a bit abrupt. I will change it in "Note: this plugin was implemented as a model plugin instead of a sensor plugin." or something similar.